ingestion/src/metadata/ingestion/source/database/databricks/metadata.py

```diff
@@ -292,13 +294,60 @@ def get_table_names(
     return [table for table in tables if table not in views]
 
 
+def get_table_type(connection, schema, table):
+    """get table type (regular/foreign)"""
+    try:
+        query = DESCRIBE_TABLE_EXTENDED.format(schema_name=schema, table_name=table)
+        rows = connection.execute(query)
+        for row in rows:
+            row_dict = dict(row)
+            if row_dict.get("col_name") == "Type":
+                return row_dict.get("data_type")
+    except Exception:
+        pass
+    return
+
+
+def get_table_columns(self, connection, table_name, schema):
+    """get databricks table columns"""
+    full_table = table_name
+    table_type = None
+    if schema:
+        full_table = schema + "." + table_name
```

Comment:
instead of creating a new variable `full_table` we can just use the `table_name` here.
`table_name = schema + "." + table_name`